sdk/lib/cmlib/hivewrt.c

```diff
@@ -188,6 +192,8 @@ HvpWriteLog(
          * FIXME: We should rather use RtlSetBits but that crashes
          * the system with a bugckeck. So for now mark blocks manually
          * by hand.
+         *
+         * TK: Maybe use Ptr[BlockIndex / 8] |= (1UL << (BlockIndex & 7));?
```

Comment:
Makes the code less readable tbh.

Reply:
It's not about readability, but about correctness. This is a bit, the other thing is a byte. Anyway, Thomas has a correct fix and this change isn't needed.